_posts/programme/2020-08-19-kickoff.md

```diff
@@ -14,42 +14,26 @@ registration_url: https://docs.google.com/forms/d/e/1FAIpQLSetlGKtfCxxaAjtHEGxaQ
 meeting_url: https://zoom.us/j/95982897830
 last_modified_at: 2020-09-02
 ---
-To kick our series off, we’re delighted to announce that the SORSE Launch
-Event will take place 13:00-15:00 UTC on Wednesday 2nd September. We’re
-pleased to be welcoming two keynote speakers at this event:
+To kick off our series off, we have run the SORSE Launch
+Event on Wednesday 2nd September. We had
+two keynote speakers at this event:
 
 - Dr Mariann Hardey, Associate Professor, Durham University: _Switching off the label "women in tech"_
 - Dr Kari Jordan, Executive Director, The Carpentries: _I wanna dance with somebody_
 
-<div>
-    {% include registration-button.html %}
-    {% include zoom-button.html %}
-    {% assign time = page.time %}
-    {% include add-to-calendar-button.html %}
-</div>
-
-The session will begin with a short welcome and introduction to SORSE followed
-by two 30 minute keynote talks. Dr Mariann Hardey will speak on the subject of
-women in tech and Dr Kari Jordan will speak about the Carpentries (both
-recorded). Further details and abstracts can be seen below. The talks will be
+The session began with a short welcome and introduction to SORSE followed
+by two 30 minute keynote talks. Dr Mariann Hardey spoke on the subject of
+women in tech and Dr Kari Jordan spoke about the Carpentries. The talks were
```

Comment:
```suggestion
by two 30 minute keynote talks. Dr Kari Jordan spoke about the Carpentries 
and Dr Mariann Hardey spoke on the subject of women in tech. The talks were
```

Reply:
Have thought about the order while working on this, but was pressed for time, so didn't change. Nice one!